### ISSUE TITLE
[Bug]:  Incompatibility of Cards on Guide and Schedule Pages with Night Mode

### ISSUE THREAD

**Issue by Pritam1136** [gabrysia694/Gym-Junkies#178]
### What happened?

Hi there! I hope you're doing well. I'd like to bring to your attention an incompatibility issue I've noticed with the night mode feature in your repository. While the night mode itself works great, there seems to be a problem with the display of cards on both the Guide and Schedule pages.

When the night mode is enabled, the cards on these pages appear to have insufficient contrast or inappropriate styling, making them difficult to read or distinguish from one another. This significantly hinders the overall user experience, especially for those who rely on the night mode for accessibility or personal preference reasons.

To resolve this issue, I suggest reviewing the styles applied to the cards on the Guide and Schedule pages when the night mode is active. Consider adjusting the text color, background color, or any other relevant styling properties to ensure better visibility and readability.

Thank you for your attention to this matter. I believe addressing this issue will greatly enhance the usability and accessibility of your repository. Please let me know if you need any additional information or if there's anything else I can do to assist you.

### Add screenshots

![Screenshot 2023-06-07 174651](https://github.com/gabrysia694/Gym-Junkies/assets/126561455/27cebd3e-3fd6-4d71-acf4-b5bddb21ae7d)

### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

**Comment by gabrysia694** [comment]
hey @Pritam1136, thank you for raising this issue. Could you suggest what excatly would you do to improve visibility and readability?

**Comment by manuel12** [comment]
@Pritam1136  @gabrysia694  For this issue  I have a proposed solution: to make the exercise title font slightly bigger and italics, this way, it helps  it stand out.

Here is how such a solution would look:

![image](https://github.com/gabrysia694/Gym-Junkies/assets/4129325/c9fc0537-96eb-4e67-8a26-dd8679c77a77)

If it's ok with both of you I can PR this.

**Comment by gabrysia694** [comment]
It looks good to me, let's see if @Pritam1136 has any more suggestions

**Comment by Pritam1136** [comment]
Hello @manuel12 

Thank you for suggesting changing the font to make the cards compatible with dark mode. However, it is crucial to consider the overall color scheme of the cards as well. In order to create a visually appealing experience in dark mode, it is recommended to modify both the font-family and the background color of the cards.

Dark mode aims to provide a comfortable reading environment by reducing eye strain. While adjusting the font-family can enhance legibility, it is equally important to choose a darker background color for the cards. By incorporating a harmonious color palette that includes darker shades and contrasting font colors, you can ensure that the cards seamlessly blend with the dark mode theme, creating an aesthetically pleasing and user-friendly design.

Thank you once again for your suggestion, and I hope this clarifies the importance of considering the overall color scheme when adapting cards for dark mode. If you have any further questions or suggestions, please feel free to reach out.

**Comment by manuel12** [comment]
@Pritam1136 I think I know what you mean, I will try some adding styling changes to the cards. If @gabrysia694 and you agree then I can make a PR about it :)

**Comment by manuel12** [comment]
Hello again @gabrysia694 @Pritam1136 

I have come up with the current implementation:

![image](https://github.com/gabrysia694/Gym-Junkies/assets/4129325/d149c7b4-426d-4fda-9fba-31a1e6c534df)

I made the exercise card be grey on dark mode, made the exercise title (also watch video text, and name of the person who added the exercise), a slightly lighter color and made the rest of the card text white.

I also left a white background on the images and added a slight border and border radius to them. 

The reason for this is that some images have transparent backgrounds and some have fixed white backgrounds. In order to have consistency I decided to have a white background in all images.

I hope this works and wait for your feedback in any case :)
